INSTALL.md

```diff
@@ -56,11 +56,15 @@ Example: In the case of stock eZ Platform 1.11 and higher that would specificall
 ```
 
 
-### Enable EzPublishLegacyBundle
+### Enable EzPublishLegacyBundle and EzSystemsEzPlatformXmlTextFieldTypeBundle
 Edit `app/AppKernel.php`, and add `new eZ\Bundle\EzPublishLegacyBundle\EzPublishLegacyBundle( $this ),`
 at the end of  the `$bundles` array. Pay close attention to the `$this` argument, LegacyBundle is a bit 
 spoiled and has high expectations from its collaborators ;)
 
+If you still plan to have content in XmlText _(as opposed to migrating to RichText straight away)_, then
+you'll also need to enable `EzSystems\EzPlatformXmlTextFieldTypeBundle\EzSystemsEzPlatformXmlTextFieldTypeBundle`
+bundle.
```

Comment:
and maybe `... bundle, which gets installed together with legacy-bridge.`

Reply:
Done.